File: Latihan2/Product.java
package Latihan2;

class Product {
    private String price;
    private String idProduct;

    // Constructor
    public Product(){
    }

    // Get-set Price
    public String getPrice(){
        return price;
    }

    public void setPrice(String price){
        this.price = price;
    }

    // Get-set ID Product
    public String getIdProduct(){
        return idProduct;
    }

    public void setIdProduct(String idProduct){
        this.idProduct = idProduct;
    }
}

File: Latihan2-php/Product.php
<?php

class Product{
    private $price;
    private $idProduct;

    public function __construct(){
    }

    // Get-set Price
    public function setPrice($price){
        $this->price = $price;
    }

    public function getPrice(){
        return $this->price;
    }

    // Get-set ID Product
    public function setIdProduct($idProduct){
        $this->idProduct = $idProduct;
    }

    public function getIdProduct(){
        return $this->idProduct;
    }
}

?>